translations/en_GB/atrium-translations-en_GB-common/src/main/kotlin/ch/tutteli/atrium/translations/DescriptionResultAssertion.kt

```diff
@@ -0,0 +1,13 @@
+package ch.tutteli.atrium.translations
+
+import ch.tutteli.atrium.assertions.DescriptiveAssertion
+import ch.tutteli.atrium.reporting.translating.StringBasedTranslatable
+
+/**
+ * Contains the [DescriptiveAssertion.description]s of the assertion functions which are applicable to [Map].
+ */
+enum class DescriptionResultAssertion(override val value: String) : StringBasedTranslatable {
+    IS_NOT_SUCCESS("!! is not a Success"),
+    VALUE("value"),
+    ILLEGAL_STATE_EXCEPTION("Illegal state exception !!")
```

Comment:
not sure why you introduced this one but I think it is unnecessary, the failure case is covered by `IS_NOT_SUCCESS`, isn't it?